datus-mysql/README.md

```
@@ -97,26 +95,89 @@ uv run pytest tests/ -v --cov=datus_mysql --cov-report=term-missing
+
+### Test Markers
 
-For more details, see [tests/integration/README.md](tests/integration/README.md).
+| Marker | Description |
+|--------|-------------|
+| `integration` | Requires a running MySQL instance |
+| `acceptance` | Core functionality tests (subset of unit + integration) |
+
+## Code Structure
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)